Frontend/DinoTracker/app/controllers/staff.ts

```diff
@@ -10,32 +10,51 @@ export default class Staff extends Controller
 
   @tracked isDisplayingErrorMessage = false;
   @tracked isShowingModal = false;
+  @tracked isPaleontologist = false;
   @tracked name = '';
   @tracked username = '';
 
   @action
   toggleCreateUserModal()
   {
     this.isDisplayingErrorMessage = false;
+    this.isPaleontologist = false;
     this.name = '';
     this.username = '';
     this.isShowingModal = !this.isShowingModal;
   }
 
+  @action
+  toggleIsPaleontologist()
+  {
+    this.isPaleontologist = !this.isPaleontologist;
+  }
+
   @action
   async attemptCreateUser()
   {
-    let variables = { name: this.name, username: this.username};
     try
     {
-      let response = await this.apolloRepository.createUser(variables);
+      let response : boolean;
+      if (this.isPaleontologist)
```

Comment:
Don't worry about changing this here but keep it in mind going forward:

Instead of using a boolean here, isPaleontologist could be an enum that represents the type. That way, if we add more user types, we've already done the work to support 3 cases. Your conditions would then change to be something more like:

```js
enum StaffType {
  Paleontologist,
  Researcher
};

@tracked creatingStaffType: StaffType;

@action
toggleIsPaleontologist() {
  if (this.creatingStaffType == StaffType.Paleontologist) {
    this.creatingStaffType = StaffType.Researcher;
  } else {
    this.creatingStaffType = StaffType.Paleontologist;
  }
}

@action
async attemptCreateUsesr() {
  ...  
  if (this.creatingStaffType == StaffType.Paleontologist) {
    ...
  } else {
    ...
  }
}
```

Then it's easy to add more staff types.